packages/server/lib/modes/run.js

```diff
@@ -1244,17 +1260,16 @@ module.exports = {
         }
       }
 
-      // Close the browser if the environment variable is set to do so
-      // if (process.env.CYPRESS_INTERNAL_FORCE_BROWSER_RELAUNCH) {
-      //   debug('attempting to close the browser')
-      //   await openProject.closeBrowser()
-      // } else {
-      debug('attempting to close the browser tab')
-      await openProject.resetBrowserTabsForNextTest(shouldKeepTabOpen)
-      // }
+      // we do not support experimentalSingleTabRunMode for e2e
+      if (testingType === 'e2e' || !config.experimentalSingleTabRunMode) {
+        debug('attempting to close the browser tab')
+
+        await openProject.resetBrowserTabsForNextTest(shouldKeepTabOpen)
 
-      debug('resetting server state')
-      openProject.projectBase.server.reset()
+        debug('resetting server state')
+
+        openProject.projectBase.server.reset()
```

Comment:
we might be dropping the server reset(). does it make single tab move if we reset this regardless of the flag?

Reply:
What do you mean "move"?

Also I don't exactly remember why we reset here at all - I'm not sure how this impacts single tab run mode uniquely, I'm guessing anything that impacts regular run mode will also impact single tab run mode.